kubernetes/samples/scripts/common/utility.sh

```diff
@@ -293,6 +303,7 @@ function createFiles {
     sed -i -e "s:%JAVA_OPTIONS%:${javaOptions}:g" ${domainPropertiesOutput}
     sed -i -e "s:%T3_CHANNEL_PORT%:${t3ChannelPort}:g" ${domainPropertiesOutput}
     sed -i -e "s:%T3_PUBLIC_ADDRESS%:${t3PublicAddress}:g" ${domainPropertiesOutput}
+    sed -i -e "s|%IMAGE_NAME%|${defaultImageName}|g" ${domainPropertiesOutput}
```

Comment:
Need check whether the image property is setting, then decide use which name. See the following snippet:
    if [ -z $image ]; then
      sed -i -e "s|%IMAGE_NAME%|${defaultImageName}|g" ${domainPropertiesOutput}
    else
      sed -i -e "s|% IMAGE_NAME%|${image}|g" ${domainPropertiesOutput}
    fi

Reply:
The proposed behaviors are
1) the Docker sample always tag the generated image with the default image name (defaultImageName)
2) Optionally if the "image" property is set, tag the generated image with the value of "image" property as well (see the comment in line 199 in create-domain.sh).

We could change it to tag with the defaultImageName only if the "image" property is set, which is probably cleaner.

Reply:
The confusing thing is that the image name in domain.properties may be different from the image name in domain.yaml

Reply:
I updated the script to tag with the default image name only if the "image" is not specified.